### DISCUSSION TITLE
Dendron frontmatter (created and updated with unix timestamp)

### DISCUSSION THREAD

**Discussion by grizz-pl** [GitJournal/GitJournal#861] [Feature Requests]
## Environment

**App version:** 1.73.2
**Device information:**
 - OS name and version:Android
 

## Description

**What you'd like to happen:** I'd like to get support for dendron Frontmatter https://www.dendron.so/notes/ffec2853-c0e0-4165-a368-339db12c8e4b.html

updated

Unix timestamp of when note was updated
created

Unix timestamp of when note was created

**Alternatives you've considered:**Abblity to sort files by file's attributes (created, last mod) not by yaml header.

**Comment by vHanda** [comment]
Related to #279

**Comment by andyli** [comment]
Looks like dendron's timestamps are in ms, e.g.
```
updated: 1653361144917
```

GitJournal updated it in seconds
```
updated: 1653364684
```

Is there a way to set GitJournal's timestamp format?
